code/game/objects/items/stacks/rods.dm

```diff
@@ -73,3 +73,12 @@ var/global/list/datum/stack_recipe/rod_recipes = list ( \
 
 /obj/item/stack/rods/cyborg/update_icon()
 	return
+
+/obj/item/stack/rods/ten
+	amount = 10
+
+/obj/item/stack/rods/twentythree

```

Comment:
This crusade against modified types is silly, in this case it's perfectly fine to adjust the amount var of the stacks, the amount var is very unlikely to go away at any point.